src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -188,6 +189,27 @@ public VulnerabilityTrend getYearlyVulnTrend(String organizationId) throws IOExc
         }
     }
 
+    /**
+     * Get all Vulnerability Trend for an Organizations.

```

Comment:
Can you update this comment to reflect how this method is different from the one above?